lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -4,6 +4,19 @@
 
 module KubernetesDeploy
   class KubernetesResource
+    def self.logger=(value)
+      @logger = value
+    end
+
+    def self.logger
+      @logger ||= begin
+        l = Logger.new($stderr)
+        l.formatter = proc do |_severity, _datetime, _progname, msg|
+          "#{msg}\n"
```

Comment:
Because this logger is for dumping output raw: 
- `[KUBESTATUS]` lines for eventual consumption by a visualization
- Pod logs from k8s (e.g. from migrations)

(white lines in pic below)

![image](https://cloud.githubusercontent.com/assets/4789493/23179209/171ffc74-f83b-11e6-8b92-9c38f624038d.png)

Perhaps it should be named `raw_logger`?